_posts/en/newsletters/2018-07-24-newsletter.md

```diff
@@ -0,0 +1,169 @@
+---
+title: 'Bitcoin Optech Newsletter #5'
+permalink: /en/newsletters/2018/07/24/
+name: 2018-07-24-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+version: 1
+---
+This week's newsletter includes news about a new method under
+development for describing which outputs you'd like to find, the merge
+of partially-signed Bitcoin transactions into Bitcoin Core, and several
+other notable Bitcoin Core merges.
+
+## Action items
+
+- Bitcoin Core [0.16.2RC2][] released for testing in preparation for a
+  maintenance release that will provide bugfixes and backports.
+  Community testing is highly appreciated.  Note, there was no RC1 due
+  to a metadata problem being detected and corrected after RC1 was
+  tagged but before it was released.
+
+## Dashboard items
+
+- Transaction fees are lower than they were this time last week.  Anyone
+  who can wait 12 or more blocks for confirmation can reasonably pay the
+  default minimum feerate.  It's a good time to [consolidate UTXOs][].
+
+- The number of [native segwit outputs][p2shinfo bech32] had been
+  increasing steadily over time, but dropped by about 400,000 (80%) this
+  week, probably as part of a consolidation by an exchange.  The average
+  number of new native segwit outputs created per hour remains
+  relatively constant, indicating no obvious decrease in adoption.
+  (Remember that if your service accepts payments to native segwit
+  addresses, you'll [save up to 16%][native segwit save] in fees when
+  you spend them later compared to using P2SH-wrapped segwit, which can
+  itself [save 26% or more][p2sh segwit save] compared to legacy
+  outputs.)
+
+## News
+
+- **Bitcoin Optech publicly announced:** we received great coverage in
+  [Bitcoin Magazine][announce bmag], [Coindesk][announce cdesk], and several
+  other publications.  This wouldn't have been possible without the
+  support of our founding sponsors and member companies.  Thank you!
+
```

Comment:
Added an item on our first workshop.

Reply:
I love it, thanks!  I'm made two small changes: (1) made "Replace-by-fee and Child-pays-for-parent, and Coin Selection" into a list and dropped the title caps, (2) made both lists in this content use the serial comma (a, b, and c) to match the style of the other lists in the newsletter (serial comma is my preferred list punctuation, but if you want to use the non-serial comma in our docs, we can make a style guide).